autogpt_platform/backend/backend/data/graph.py

```diff
@@ -431,6 +451,7 @@ async def get_graph(
     version: int | None = None,
     template: bool = False,
     user_id: str | None = None,
+    hide_secrets: bool = False,
```

Comment:
nit: rename this `hide_credentials` or `omit_credentials` so we can keep using it after moving all legacy block secrets over to the credentials system